lib/mysql_rewinder/cleaner.rb

```diff
@@ -19,7 +19,11 @@ def clean_all
 
     def clean(tables:)
       target_tables = (tables - @except_tables) & all_tables
-      return if target_tables.empty?
+
+      if target_tables.empty?
+        @logger&.debug "[MysqlRewinder][#{@db_config[:database]}] the table to be deleted is empty."
```

Comment:
I think the following log message is a more accurate representation of the code's behavior

```suggestion
        @logger&.debug { "[MysqlRewinder][#{@db_config[:database]}] Skip DELETE query because target_table is empty." }
```